daml/Marketplace/Lifecycle/Model.daml

```diff
@@ -140,7 +140,7 @@ template Effect
                 let
                   settlementId = lifecycleId <> "-" <> partyToText snapshot.deposit.account.owner <> "-Expiry"
                   asset = snapshot.deposit.asset with id.signatories = fromList [bondRegistrar, issuer]; id.version = newVersion
-                instructionCids <- createInstructions False operator payingAgent bondRegistrar snapshot.deposit.account.owner issuer settlementId 0 asset
+                instructionCids <- createInstructions False operator payingAgent bondRegistrar snapshot.deposit.account.owner bondRegistrar settlementId 0 asset
```

Comment:
<placeholder>

Reply:
Yes, that's correct. It was the pragmatic solution, as there's no need for the issuer to receive the redeemed bonds anyway.